meta_request/test/unit/meta_request/utils_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+describe MetaRequest::Utils do
+  describe ".dev_callsite" do
+    it "returns line parsed with Callsite if rails root is found in multi-line trace" do
```

Comment:
I plan to introduce Rubocop soon and default setting for the string literals is the single quote.